_stbt/types.py

```diff
@@ -12,8 +12,38 @@
 
 
 class Position(namedtuple('Position', 'x y')):
-    """A point with ``x`` and ``y`` coordinates."""
-    pass
+    """A point with ``x`` and ``y`` coordinates.
+
+    >>> Region.bounding_box(Position(1, 8), Position(7, 3))
+    Region(x=1, y=3, right=7, bottom=8)
+    >>> b = Region(4, 4, right=13, bottom=10)
+    >>> b.contains(Position(1, 3))
+    False
+    >>> b.contains(Position(7, 8))
+    True
+    >>> b.contains(Position(13, 10))
+    True
```

Comment:
This should be False, because Region is a half-open interval `[x, right)` and `[y, bottom)`.